scss/_patterns_contextual-menu.scss

```diff
@@ -63,11 +63,9 @@
     }
   }
 
-  .p-contextual-menu__toggle {
-    margin-bottom: $spv-inner--small + ($spv-nudge-compensation * 2);
-  }
-
-  .p-contextual-menu__dropdown {
+  // compensate for bottom margin when toggle is a button
+  button.p-contextual-menu__toggle + .p-contextual-menu__dropdown, // sass-lint:disable-line no-qualifying-elements
```

Comment:
@anthonydillon How do you feel about this?

It seems it's the only way, because we need to address all the buttons (`<button>` without class and with `p-button`). 

Unless we would just introduce yet another class, for buttons with contextual menu specifically (`p-contextual-menu__toggle--button`). Which we would do just to avoid using `button` in selector.